bin/storm.py

```diff
@@ -263,6 +263,17 @@ def upload_credentials(*args):
         jvmtype="-client",
         extrajars=[USER_CONF_DIR, STORM_BIN_DIR])
 
+def heartbeats(*args):
+    """Syntax: [storm heartbeats [cmd]]
+
+    list [KEY...] - lists heartbeats nodes under KEY currently in pacemaker.
```

Comment:
It might be more intuitive to use `PATH` here instead of `KEY`.  We use it by running `storm heartbeats /`, etc.